### DISCUSSION TITLE
Difference between the apparent diameter measurements shown in the information of the selected objects and the one shown in the astronomical calculations window

### DISCUSSION THREAD

**Discussion by tmsantiago** [Stellarium/stellarium#2744] [General]
Hi guys, I was looking at the apparent diameter of the Moon for the next few days so i decided to produce a graph of angular size versus time for 1 month and 1 hour step in the Graphs Tab in the Astronomical Calculations window. I decided to compare one of the points on the graph with the measure of the apparent diameter that is shown on the main screen when I select the Moon. I inspected the point equivalent to the date and time when I was viewing the Moon and noticed that the values were different. Is there any explanation for this difference?

Thanks for advanced,

**Comment by alex-w** [comment]
Looks like bug

**Reply by tmsantiago** [reply]
Hi Alexander! Thanks for your answer. 
I'm using Stellarium 1.0 Version 1.22.3, Based on Qt 6.2.4 in Linux Mint 20.3 Cinnamon.

**Reply by tmsantiago** [reply]
> Looks like bug

When I took the current distance from the Earth in km and half of Moon's diameter in km (shown on screen when The Moon is selected ) to manually calculate the angular diameter, I found the same value which is shown field "apparent size" in the main screen, but this value is higher than that shown in the point of graph angular size versus time for the same date and time. Please, could you tell me what is the database used to generate that graph?
Many thanks,

**Reply by alex-w** [reply]
@gzotti please check lines [6264](https://github.com/Stellarium/stellarium/blob/dafba93fa310b959e8b2a43b13373f005c8de4b2/src/gui/AstroCalcDialog.cpp#L6264) and [6290-6291](https://github.com/Stellarium/stellarium/blob/master/src/gui/AstroCalcDialog.cpp#L6290) - why we have application of UTCShift twice and why we mark it as UTC?
